packages/vitest/src/utils/source-map.ts

```diff
@@ -25,10 +25,14 @@ export async function interpretSourcePos(stackFrames: ParsedStack[], ctx: Vitest
   for (const frame of stackFrames) {
     if ('sourcePos' in frame)
       continue
-    const transformResult = ctx.server.moduleGraph.getModuleById(frame.file)?.ssrTransformResult
-    if (!transformResult)
+    const ssrTransformResult = ctx.server.moduleGraph.getModuleById(frame.file)?.ssrTransformResult
+    const transformResult = ctx.server.moduleGraph.getModuleById(frame.file)?.transformResult
+    const result = ssrTransformResult || transformResult
```

Comment:
Why do we need this, if it's not used later? Maybe you wanted to put it on L#34?

Reply:
It's only used in the check `if (!result) continue`
The rationale behind it is that there is no need to calculate the position if the file hasn't been transformed.

And sometimes, the file has been transformed but `ssrTransformResult` is unedfined while `transformResult` isn't.

Reply:
The reason I didn't put it in line `#34` is that it has the wrong sourcemap. 

I think it's the sourcemap of transforming files to run in vite-node context where imports are mapped differently, and that's not the only one, so it has different sourcemaps.

Reply:
But it means `map` on L35 can be `undefined`? I don't really see a point in this check, tbh. It makes more sense to `continue` on L34, if map is not defined.

Reply:
Makes sense @sheremet-va Done 👍